lib/src/delegates/asset_picker_text_delegate.dart

```diff
@@ -844,3 +845,100 @@ class FrenchAssetPickerTextDelegate extends AssetPickerTextDelegate {
   @override
   String get sUnitAssetCountLabel => 'quantité';
 }
+
+/// Added Turkish Language Support
+///
+/// @date 20220601
+/// @author Yusuf Gündoğdu
+/// @author Cevheri
+class TurkishAssetPickerTextDelegate extends AssetPickerTextDelegate {
+  const TurkishAssetPickerTextDelegate();
+
+  @override
+  String get languageCode => 'tr';
+
+  @override
+  String get confirm => 'Onayla';
+
+  @override
+  String get cancel => 'İptal';
+
+  @override
+  String get edit => 'Düzenle';
+
+  @override
+  String get gifIndicator => 'GIF';
+
+  @override
+  String get loadFailed => 'Yükleme başarısız!';
+
+  @override
+  String get original => 'Orijinal';
+
+  @override
+  String get preview => 'Önizleme';
+
+  @override
+  String get select => 'Seç';
+
+  @override
+  String get emptyList => 'Boş liste';
+
+  @override
+  String get unSupportedAssetType => 'Desteklenmeyen HEIC medya tipi!';
```

Comment:
`!` seems to be unnecessary here, same as other fields with `!`.

Reply:
Completed.